pkg/report/render.go

```diff
@@ -46,12 +49,20 @@ func NewHTMLRenderer() (*HTMLRenderer, error) {
 		_, ok := m[k]
 		return ok
 	}
+	yamlFormat := func(m map[string]any) string {
+		yaml, err := yaml.Marshal(m)
+		if err != nil {
+			log.Println(err)
```

Comment:
```suggestion
			return err.String()
```

Let's show the error in the report if there is such